File: EvenNumbers.vb
Public Class EvenNumbers

    Dim first As Integer
    Dim last As Integer
    Dim result As New List(Of Integer)

    ' Simulate a constructor
    Public Sub _init_(alpha As Integer, omega As Integer)
        first = alpha
        last = omega
    End Sub

    ' Returns an list Of the desired Set Of even numbers
    Public Function prepResult() As List(Of Integer)
        ' Loop from start to stop and rip out even numbers;

        Dim i = 0
        For i = first To last
            If i Mod 2 = 0 Then ' modulo(%) is explained later
                result.Add(i)
            End If
        Next i

        Return result
    End Function

End Class

File: EvenNumbers.py
# Sure enough this is a module

# define a class
class EvenNumerals:
    
    ###
     # Our constructor.
     # @param alpha - for the start value
     # @param omega - for the end value
     ##
    def __init__(self, alpha, omega):
        self.start = alpha
        self.stop = omega
        self.result = [] # list to hold our answers

    # Returns an list of the desired set of even numbers
    def prepResult(self):
        # Loop from start to stop and rip out even numbers;
        self.i = 0
        while self.start <= self.stop:
            if self.start % 2 == 0: # modulo(%) is explained later
                self.result.append(self.start)
                self.i += 1
                
            self.start = self.start + 1 # increase start by 1
            
        return self.result

